metabase/metabase
# Fix: Link card spacing isn't quite right on the new grid

Closes https://github.com/metabase/metabase/issues/31701

### Description

Makes sure that all links are vertically aligned within the smaller grid.
Converts this:
<img width="204" alt="image" src="https://github.com/metabase/metabase/assets/25306947/768b9eb7-a433-446b-8572-70e4f3a9c45d">

to this:
<img width="196" alt="image" src="https://github.com/metabase/metabase/assets/25306947/fc00a530-7954-4334-becf-5198ccf84b6d">


### How to verify

1. Create a new dashboard
2. Add a link, whether it's a link to an entity in Metabase or a custom link
3. The text should be vertically aligned

### Checklist
